data/styles/default/include/priorities_4_overlays.prio.txt

```diff
@@ -713,6 +713,7 @@ highway-path-tunnel                                 # pathtext z15- (also has li
 === 2820
 
 highway-steps                                       # pathtext z16- (also has line z15-)
+highway-ladder
```

Comment:
It'd be better to have this at a much lower priority (`850`) as we don't want it displaying instead of POI etc.